.github/workflows/push_image_to_dockerhub.yml

```
@@ -0,0 +1,51 @@
+# Build the new Docker image on every commit to master/develop branch and on every new tag.
+# No caching is involved for the image build. The new image is then pushed to the Docker Hub.
+
+name: build-and-push-to-dockerhub
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+    tags:
+      - 'v*'
```

```suggestion
      - "v[0-9]+.[0-9]+.[0-9]+*"
```

this is what we use for the other release workflow